### ISSUE TITLE
PIT loss returns wrong loss

### ISSUE THREAD

**Issue by RahulDasari** [asteroid-team/asteroid#704]
I'm trying to train using PIT loss and i get a loss of 80 when i pass identical tensors(batch , src, time) as source and est_source. Shouldn't it be zero? 

loss_func = PITLossWrapper(pairwise_neg_sisdr, pit_from='pw_mtx')
print(loss_func(torch.zeros(1,5,624),torch.zeros(1,5,624)))
>>>80

**Comment by mpariente** [comment]
Please explain how you solved your issue, so future users can discover the solution.
